src/main/java/com/egen/controller/OrderController.java

```diff
@@ -1,50 +1,95 @@
+     * @param id accepts Order id as input
+     * @return Order object
+     */
+    @RequestMapping(method = RequestMethod.GET,value = "/{id}",produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+    public Orders getOrderById(@PathVariable("id")Long id){
+        return ordersService.findOne(id);
     }
 
-    public ResponseEntity<List<Order>> getAllOrdersWithInInterval(ZonedDateTime startTime, ZonedDateTime endTime){
-        //TODO
-        return null;
+    /**
+     * Fetches all the records of orders places between the given interval
+     * @param startTime - Time of the order placed
+     * @param endTime - Time of the order placed
+     * @return List of orders
+     */
+    @RequestMapping(method = RequestMethod.GET,value = "/interval",produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+    public List<Orders> getAllOrdersWithInInterval(@RequestParam(name = "startTime") Timestamp startTime,
+                                                   @RequestParam(name = "endTime") Timestamp endTime){
+        return ordersService.findWithinInterval(startTime,endTime);
     }
 
-    public ResponseEntity<List<Order>> top10OrdersWithHighestDollarAmountInZip(String zip){
-        //TODO
-        return null;
+    /**
+     * Fetches the top 10 highest amount in the given area
+     * @param zip - Zipcode of the address
+     * @return List of orders
+     */
+    @RequestMapping(method = RequestMethod.GET,value = "zipcode/{zip}",produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+    public List<Orders> top10OrdersWithHighestDollarAmountInZip(@PathVariable("zip") String zip){
+        return ordersService.findTop10OrdersWithHighestDollarAmountInZip(zip);
     }
 
-    public ResponseEntity<Order> placeOrder(Order order){
-        return null;
+    @RequestMapping(method = RequestMethod.POST,value = "/placeorder",consumes = MediaType.APPLICATION_JSON_UTF8_VALUE,
+            produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+    public void placeOrder(@RequestBody InputData inputData){
+        System.out.println(inputData.toString());
+        ordersService.createOrder(inputData);
     }
 
-    public ResponseEntity<Order> cancelOrder(Order order){
-        return null;
+    /**
+     * This method will cancel the selected order
+     * @param order -  orderStatus = "CANCELLED"
+     * @param id - Order id
+     * @return
+     */
+    @RequestMapping(method = RequestMethod.PUT,value = "/cancel/{id}",consumes = MediaType.APPLICATION_JSON_UTF8_VALUE,
+            produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+    public Orders cancelOrder(@RequestBody Orders order,@PathVariable("id")Long id){
+        return ordersService.cancelOrder(order,id);
     }
 
-    public ResponseEntity<Order> updateOrder(Order order){
-        return null;
+    /**
+     * Updates the selected order by changing the values desired
+     * Currently this method is used to change the order status to delivered
+     * @param id - order id
+     * @return
+     */
```

Comment:
Again good job with comments and documentation. @requestmapping can be replaced with @get/@put/@post